test/battle/hold_effect/covert_cloak.c

```diff
@@ -162,3 +167,15 @@ SINGLE_BATTLE_TEST("Covert Cloak blocks Sparkling Aria in singles")
         }
     }
 }
+
+SINGLE_BATTLE_TEST("Covert Cloak does not prevent ability stat changes")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET) { Item(ITEM_COVERT_CLOAK); }
+        OPPONENT(SPECIES_ELDEGOSS) { Ability(ABILITY_COTTON_DOWN); }
+    } WHEN {
+        TURN { MOVE(player, MOVE_TACKLE); }
+    } SCENE {
+        NOT MESSAGE("Wobbuffet's Speed won't go lower!");
```

Comment:
```suggestion
        MESSAGE("Wobbuffet's Speed fell!");
```
Wrong thing being checked

Reply:
see comment above